.github/workflows/sdk_build.yml

```diff
@@ -27,17 +27,22 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: 3.1.301
+      - name: Test solution - release
+        run: |
+          unset GITHUB_ACTIONS #disable deterministic builds, just for test run. Deterministic builds break coverage for some reason
```

Comment:
is this only for dotnet or does this need to be added for other repos also?

Reply:
There's an issue with the code coverage framework and deterministic builds
I tried the mitigation suggested in:
https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/DeterministicBuild.md
but I couldn't get it to work correctly. This was the simplest mitigation that I could get to work.

Reply:
I had some discussion about this offline. I'm going to take a look and see if I can figure out a better fix in a short amount of time. If not then we'll ship this.